e2e-tests/gef/cypress/integration/review-ukef-approval-with-issued-and-unissued-facilities-MIA-and-resubmit-to-ukef.spec.js

```diff
@@ -438,9 +438,9 @@ context('Submit to UKEF', () => {
       applicationPreview.submitButton().should('exist');
       applicationPreview.submitButton().click();
       applicationSubmission.submitButton().click();
-      applicationSubmission.confirmationPanelTitle().contains(`${toTitleCase(CONSTANTS.DEAL_SUBMISSION_TYPE.MIA)} submitted to UKEF`);
+      applicationSubmission.confirmationPanelTitle().contains(`${toTitleCase(CONSTANTS.DEAL_SUBMISSION_TYPE.MIN)} submitted to UKEF`);
```

Comment:
the file has MIA in the title - should one of these be changed?

Reply:
ah I see, could we add in the test something like 

"it should change from MIA to MIN"

Reply:
done